DuckDuckGoTests/URLExtensionTests.swift

```diff
@@ -20,11 +20,36 @@
 import XCTest
 
 class URLExtensionTests: XCTestCase {
```

Comment:
Doesn't appear to be any test cases that don't use a scheme (even though the code deals with that scenario).